certs_to_crlite.py

```diff
@@ -0,0 +1,49 @@
+import censys.certificates
+from decouple import config
+from filter_cascade import FilterCascade
+import sys
+
+CENSYS_API_UID = config("CENSYS_API_UID", None)
+CENSYS_API_SECRET = config("CENSYS_API_SECRET", None)
+
+if not CENSYS_API_UID or not CENSYS_API_SECRET:
+    print "Must set CENSYS_API_UID and CENSYS_API_SECRET"
+    sys.exit(1)
+
+certificates = censys.certificates.CensysCertificates(
+    CENSYS_API_UID, CENSYS_API_SECRET
+)
+
+MLBF_FILENAME = 'moz-crlite-mlbf'
+
+a = []
+b = []
+
+certs_list = certificates.search(
+    'parsed.issuer.organization.raw: "DigiCert Inc"'
+)
+
+for idx, c in enumerate(certs_list):
+    print "Iteration %i" % idx
+    if idx == 500:
```

Comment:
How many entries are in the full set?

Reply:
Not sure yet, but according to the amazing new [Merkle Town tool from Cloudflare](https://ct.cloudflare.com/), there are 10M pre-certs in the largest CT log. (Google Pilot) But I'm not sure I understand if revocations are included in CT logs, or if we have to crawl CT log data to look up revocations?